### ISSUE TITLE
ibm_resource_key and Databases for Postgresql

### ISSUE THREAD

**Issue by kirktso** [IBM-Cloud/terraform-provider-ibm#456]
### Terraform Version

```sh
❯ terraform -version
Terraform v0.11.10
+ provider.ibm (unversioned)
+ provider.local v1.1.0
```

provider.ibm is not reporting its version here. It is 0.13.0.

### Affected Resource(s)

`ibm_resource_key.postgresql`

### Terraform Configuration Files

```hcl
resource "ibm_resource_instance" "postgresql-us-south-dev-sh-too" {
  name              = "PostgreSQL US South Dev SH Too"
  service           = "databases-for-postgresql"
  plan              = "standard"
  location          = "us-south"
  resource_group_id = "${data.ibm_resource_group.group.id}"
}

resource "ibm_resource_key" "postgresql-us-south-dev-sh-too" {
  name                  = "postgresql-us-south-dev-sh-too"
  role                  = "Operator"
  resource_instance_id  = "${ibm_resource_instance.postgresql-us-south-dev-sh-too.id}"
}
```

### Debug Output

Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

### Panic Output

If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior

There should not be an error.

### Actual Behavior

The following error is shown during `terraform apply`:

```text
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  + ibm_resource_key.postgresql-us-south-dev-sh-too
      id:                   <computed>
      credentials.%:        <computed>
      name:                 "postgresql-us-south-dev-sh-too"
      parameters.%:         <computed>
      resource_instance_id: "crn:v1:bluemix:public:databases-for-postgresql:us-south:a/525a68e8dc23379b3c04421286ea18b3:59603702-4d32-4a47-bf88-d0842d03e98d::"
      role:                 "Operator"
      status:               <computed>


Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

ibm_resource_key.postgresql-us-south-dev-sh-too: Creating...
  credentials.%:        "" => "<computed>"
  name:                 "" => "postgresql-us-south-dev-sh-too"
  parameters.%:         "" => "<computed>"
  resource_instance_id: "" => "crn:v1:bluemix:public:databases-for-postgresql:us-south:a/525a68e8dc23379b3c04421286ea18b3:59603702-4d32-4a47-bf88-d0842d03e98d::"
  role:                 "" => "Operator"
  status:               "" => "<computed>"

Error: Error applying plan:

1 error(s) occurred:

* ibm_resource_key.postgresql-us-south-dev-sh-too: 1 error(s) occurred:

* ibm_resource_key.postgresql-us-south-dev-sh-too: unexpected EOF
```

The resource key is created every time I run `terraform apply`. However,
every time I run it, I get the above message - even though it is already
out there multiple times.

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

- This stopped worked with no changes to terraform configuration
  files last week.
- I set up a new PostgreSQL database from scratch and tried to add the resource key.
- Any resoure key being applied to our database seems to fail
- I have tried removing the resource_key from the Terraform state and
  then tried using the resource_key as a datasource. It also fails.
- Just completely guessing, I think whatever API call that fetches the
  resource key fails. It creates it, but has no idea it was created and
  it fails just trying to read the data as a datasource.
- Since this is new thing that started 2 or 3 days ago, is it possible
  the API for this resource has changed?
- I am using v.0.13.0
- This may be a duplicate of #451. I am submitting this because I wrote this all up already ;-). Also wanted to make sure this was looked at for Postgresql, in case there is a difference.

### References

N/A

**Comment by hkantare** [comment]
The fix of #451 resolves this issues...The fix is available in master

**Comment by kirktso** [comment]
Thank you @hkantare. This seems to be working now.
